src/javascript/app_2/Modules/Trading/Components/Form/TradeParams/allow_equals.jsx

```diff
@@ -0,0 +1,31 @@
+import React          from 'react';
```

Comment:
Since it's used only as part of `Amount` component, maybe we can have this file structure for `allow_equals` component?
    ├── Modules/
    │   ├── Trading/
    │   │   ├── Components/
    │   │   │   └── Form/
    │   │   │       ├── TradeParams/
    │   │   │           ├── Amount/ <--- here
You can refer to other components like `MessageBox` as an example.

Reply:
is the others in the trade params need to be restructured as well